crates/rolldown_testing/src/test_config/mod.rs

```diff
@@ -35,8 +35,8 @@ pub struct TestConfig {
   /// If `true`, the fixture are expected to fail to compile/build.
   pub expect_error: bool,
   #[serde(default, rename = "_comment")]
-  /// An workaround for writing comments in JSON.
-  pub _comment: String,
+  /// A workaround for writing comments in JSON.
+  pub comment: String,
```

Comment:
Please don't rename `_comment` to `comment`.

Reply:
error: field marked as public but also inferred as unused because it's prefixed with `_`
  --> crates/rolldown_testing/src/test_config/mod.rs:39:3
   |
39 |   pub _comment: String,
   |   ^^^^^^^^^^^^
   |
   = help: consider removing the underscore, or making the field private
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#pub_underscore_fields
   = note: `-D clippy::pub-underscore-fields` implied by `-D clippy::pedantic`
   = help: to override `-D clippy::pedantic` add `#[allow(clippy::pub_underscore_fields)]`

Reply:
Use `#[allow(clippy::pub_underscore_fields)]` for now.

Reply:
fixed